packages/web/src/common/store/profile/sagas.js

```diff
@@ -189,13 +198,21 @@ export function* fetchSolanaCollectiblesForWallets(wallets) {
 }
 
 export function* fetchSolanaCollectibles(user) {
-  const apiClient = yield getContext('apiClient')
+  const getSDK = yield getContext('audiusSdk')
```

Comment:
almost wrote `yield*` but we're not in typescript land 👻 😨 

I take it you can't use the `getSDK` helper here?

Reply:
I have refreshed my knowledge and replaced this with direct calls to `getSDK()`.

TIL that typed-redux-saga is not just type decorators, but actually replaces all of the effect creator functions (which return objects, by definition not iterable by `yield*`) with generator functions that wrap the effect creators and use plain `yield` internally! That was the piece that used to confuse me so much.